solc/CommandLineInterface.cpp

```diff
@@ -1563,7 +627,7 @@ bool CommandLineInterface::processInput()
 
 		if (!successful)
 		{
-			if (m_args.count(g_argErrorRecovery))
+			if (m_options.errorRecovery)
```

Comment:
You'll find a lof of such code here unfortunately. As I said above, I resisted fixing it in this PR because there's just too much of it and I wanted to focus on the split here to keep the diff as readable as possible.

I'll fix the ones you pointed out but any bigger changes I'd like to leave for future PRs.